src/components/graphqlFormGenerator/FormInput.vue

```diff
@@ -110,12 +104,12 @@ export default {
       // const ofType = this.gqlType.ofType
 
       var componentProps
```

Comment:
```suggestion
      let componentProps
```
 (but ... not actually a change on this branch)